Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,12 +1,14 @@
-public protocol GraphQLOperation: class {
+public protocol GraphQLOperationBase {
```

Comment:
We may want to keep the `: class` because it makes code using it [slightly more efficient](https://github.com/apple/swift/blob/master/docs/OptimizationTips.rst#advice-mark-protocols-that-are-only-satisfied-by-classes-as-class-protocols), and there is no reason to have structs conform to this protocol.

Reply:
Interesting